### PR TITLE
Stop using `gcloud preview` - it is obsolete

### PR DESCRIPTION
**Pull request by thockin**
Switch to supported APIs. Update docs.

@RichieEscarez because mostly docs.  We should cherry pick this.

### CODE REVIEW

File: docs/admin/cluster-management.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ The initial values of the autoscaler parameters set by ``kube-up.sh`` and some m
 or using gcloud CLI:
 
 ```
-gcloud preview autoscaler --zone compute-zone <command>
+gcloud alpha compute autoscaler --zone $ZONE <command>
```
Comment (RichieEscarez) [inline docs/admin/cluster-management.md]: ```--zone compute-zone``` is in the code block above but changed here, should they match?

Comment (RichieEscarez) [inline docs/admin/cluster-management.md]: Also, here $ZONE assumes that the environment variable is set (is that correct?) but in this topic and after searching against the repo, we dont specify how to set that variable.

File: docs/getting-started-guides/gce.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ If you want to use custom binaries or pure open source Kubernetes, please contin
 
 1. You need a Google Cloud Platform account with billing enabled. Visit the [Google Developers Console](http://cloud.google.com/console) for more details.
 1. Install `gcloud` as necessary. `gcloud` can be installed as a part of the [Google Cloud SDK](https://cloud.google.com/sdk/).
-1. Then, make sure you have the `gcloud preview` command line component installed. Run `gcloud preview` at the command line - if it asks to install any components, go ahead and install them. If it simply shows help text, you're good to go. This is required as the cluster setup script uses GCE [Instance Groups](https://cloud.google.com/compute/docs/instance-groups/), which are in the gcloud preview namespace. You will also need to **enable [`Compute Engine Instance Group Manager API`](https://developers.google.com/console/help/new/#activatingapis)** in the developers console.
+1. Enable **[`Compute Engine Instance Group Manager API`](https://developers.google.com/console/help/new/#activatingapis)** in the [Google Cloud developers console](https://console.developers.google.com).
```
Comment (RichieEscarez) [inline docs/getting-started-guides/gce.md]: We should remove the highlighting around Compute Engine Instance Group Manager API (and reserve the `` codeblocks for code examples and the bold for UI elements).

**Comment by RichieEscarez** [comment]
LGTM pending he resolution of my questions and requested formatting change.

File: docs/admin/cluster-management.md (hunk 1 above)
Comment (thockin) [inline docs/admin/cluster-management.md]: The implication here is that the user has to specify which zone they want.  I think `$ZONE` is more obviously a variable than `compute-zone`

File: docs/getting-started-guides/gce.md (hunk 2 above)
Comment (thockin) [inline docs/getting-started-guides/gce.md]: will do

**Comment by thockin** [comment]
New push is up, PTAL

File: docs/admin/cluster-management.md (hunk 1 above)
Comment (RichieEscarez) [inline docs/admin/cluster-management.md]: apparently this is the only file that still refers to ```compute-zone```, and all others use ```--zone $ZONE``` so its good to clean these out of the k8s library

Comment (thockin) [inline docs/admin/cluster-management.md]: ACK.

**Comment by thockin** [comment]
Rebased and pushed and self-lgtm'ed

**Comment by wojtek-t** [comment]
@thockin - it seems that you need to rebase again

**Comment by thockin** [comment]
rebased

**Comment by wojtek-t** [comment]
@k8s-bot test this please

**Comment by wojtek-t** [comment]
@k8s-bot test this please

**Comment by wojtek-t** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this

**Pull request merged**